roles/kubernetes/node/templates/kubelet.rkt.service.j2

```diff
@@ -62,7 +62,8 @@ ExecStart=/usr/bin/rkt run \
                 $KUBELET_ARGS \
                 $DOCKER_SOCKET \
                 $KUBELET_REGISTER_NODE \
-                $KUBELET_NETWORK_PLUGIN
+                $KUBELET_NETWORK_PLUGIN \
+                $KUBELET_CLOUDPROVIDER
```

Comment:
This is kubelet start option, not rkt. 
We set this variable in `kubelet.env` -- https://github.com/kubernetes-incubator/kargo/blob/master/roles/kubernetes/node/templates/kubelet.j2#L45 and later use it for the docker version of kubelet --
https://github.com/kubernetes-incubator/kargo/blob/master/roles/kubernetes/node/templates/kubelet.docker.service.j2#L25
It was missed in the rkt version which causes the cloud provider functionality to not work properly with rkt based kubelet.